pkg/daemon/spawn_windows.go

```diff
@@ -15,11 +15,18 @@ const (
 	daemonCreationFlags = CREATE_BREAKAWAY_FROM_JOB | CREATE_NEW_PROCESS_GROUP | DETACHED_PROCESS
 )
 
-func procAttrForSpawn() *os.ProcAttr {
+func procAttrForSpawn(stdout *os.File) *os.ProcAttr {
+	// The daemon should not inherit a reference to the stdin file descriptor
+	// of the original shell. The daemon shouldn't read anything from stdin so
+	// use the null device.
+	devnull, _ := os.OpenFile("NUL:", os.O_RDONLY, 0)
+	env := []string{
+		"SystemRoot=" + os.Getenv("SystemRoot"), // this is needed for net.Listen for some reason
+	}
```

Comment:
I made that change because the literal is non-trivial (unlike in *spawn_unix.go*) and I was already touching the adjacent lines. But I don't feel strongly enough to argue for keeping this aspect of the P.R. so I've reverted it.